tests/common/fixtures/conn_graph_facts.py

```diff
@@ -1,17 +1,33 @@
 import pytest
 import os
-import json
+import yaml
 
 @pytest.fixture(scope="module")
 def conn_graph_facts(duthost, localhost):
-    conn_graph_facts = dict()
+    conn_graph_facts = get_graph_facts(duthost, localhost, duthost.hostname)
+    return conn_graph_facts
+
+  
+@pytest.fixture(scope="module")
+def fanout_graph_facts(localhost, duthost, conn_graph_facts):
+    fanout_host = conn_graph_facts["device_conn"]["Ethernet0"]["peerdevice"]
```

Comment:
We have setups where a DUT is connected to multiple fanouts.

Reply:
I believe that this is out of scope of this PR, because main goal of this changes was adding support of Onyx based fanout. I would like to have it merged first and the we can take multiple fanout support as another PR since we don't have such configurations and I'm not able to verify it.

Reply:
Sure. I can add in those changes

Reply:
Thank you